test/DotNetty.Handlers.Proxy.Tests/ProxyHandlerTest.cs

```diff
@@ -0,0 +1,704 @@
+        private static readonly string BAD_PASSWORD = "badPassword";
+
+        internal static readonly IEventLoopGroup Group = new DefaultEventLoopGroup(3);
+
+        private static readonly ProxyServer DeadHttpProxy = new HttpProxyServer(false, TestMode.Unresponsive, null);
+        private static readonly ProxyServer InterHttpProxy = new HttpProxyServer(false, TestMode.Intermediary, null);
+        private static readonly ProxyServer AnonHttpProxy = new HttpProxyServer(false, TestMode.Terminal, DESTINATION);
+
+        private static readonly ProxyServer HttpProxy =
+            new HttpProxyServer(false, TestMode.Terminal, DESTINATION, USERNAME, PASSWORD);
+
+        private static readonly ProxyServer DeadHttpsProxy = new HttpProxyServer(true, TestMode.Unresponsive, null);
+        private static readonly ProxyServer InterHttpsProxy = new HttpProxyServer(true, TestMode.Intermediary, null);
+        private static readonly ProxyServer AnonHttpsProxy = new HttpProxyServer(true, TestMode.Terminal, DESTINATION);
+
+        private static readonly ProxyServer HttpsProxy =
+            new HttpProxyServer(true, TestMode.Terminal, DESTINATION, USERNAME, PASSWORD);
+
+        /*
+        static readonly ProxyServer deadSocks4Proxy = new Socks4ProxyServer(false, TestMode.UNRESPONSIVE, null);
+        static readonly ProxyServer interSocks4Proxy = new Socks4ProxyServer(false, TestMode.INTERMEDIARY, null);
+        static readonly ProxyServer anonSocks4Proxy = new Socks4ProxyServer(false, TestMode.TERMINAL, DESTINATION);
+        static readonly ProxyServer socks4Proxy = new Socks4ProxyServer(false, TestMode.TERMINAL, DESTINATION, USERNAME);
+
+        static readonly ProxyServer deadSocks5Proxy = new Socks5ProxyServer(false, TestMode.UNRESPONSIVE, null);
+        static readonly ProxyServer interSocks5Proxy = new Socks5ProxyServer(false, TestMode.INTERMEDIARY, null);
+        static readonly ProxyServer anonSocks5Proxy = new Socks5ProxyServer(false, TestMode.TERMINAL, DESTINATION);
+        static readonly ProxyServer socks5Proxy =
+                new Socks5ProxyServer(false, TestMode.TERMINAL, DESTINATION, USERNAME, PASSWORD);*/
+
+        private static readonly IEnumerable<ProxyServer> AllProxies = new[]
+        {
+            DeadHttpProxy, InterHttpProxy, AnonHttpProxy, HttpProxy,
+            DeadHttpsProxy, InterHttpsProxy, AnonHttpsProxy, HttpsProxy
+            //deadSocks4Proxy, interSocks4Proxy, anonSocks4Proxy, socks4Proxy,
+            //deadSocks5Proxy, interSocks5Proxy, anonSocks5Proxy, socks5Proxy
+        };
+
+        // set to non-zero value in case you need predictable shuffling of test cases
+        // look for "Seed used: *" debug message in test logs
+        private static readonly int ReproducibleSeed = 0;
+
+        public ProxyHandlerTest()
+        {
+            ClearServerExceptions();
+        }
+        
+        [Theory]
+        [MemberData(nameof(CreateTestItems))]
```

Comment:
https://dev.azure.com/SpanNetty/SpanNetty/_build/results?buildId=340&view=ms.vss-test-web.build-test-results-tab&runId=1576&resultId=107667&paneView=debug

and also test fail locally on Windows

Reply:
fixed